tests/test_unit_file_metadata_init.cpp

```diff
@@ -28,33 +28,75 @@ std::string getTestFileMatchDir()
   return srcLocation;
 }
 
+void replaceStrAll(std::string &stringToReplace,
+                   std::string const &oldValue,
+                   std::string const &newValue) {
+  size_t oldValueLen = oldValue.length();
+  size_t newValueLen = newValue.length();
+  if (0 == oldValueLen) {
+    return;
+  }
+
+  size_t index = 0;
+  while (true) {
+    /* Locate the substring to replace. */
+    index = stringToReplace.find(oldValue, index);
+    if (index == std::string::npos) break;
+
+    /* Make the replacement. */
+    stringToReplace.replace(index, oldValueLen, newValue);
+
+    /* Advance index forward so the next iteration doesn't pick it up as well. */
+    index += newValueLen;
+  }
+}
+
+std::vector<std::string> getListOfTestFileMatchDir()
+{
+  std::vector<std::string> listOfTestDirs; 
+  #ifdef _WIN32
+  std::string srcLocation = getTestFileMatchDir();
+  replaceStrAll(srcLocation,"\\","/");
+  // Directory path looks like C:/Users/vreddy/App/Temp/adkf-kasdfk-adsfl/
+  // Testing forward slash for put/get support
+  listOfTestDirs.push_back(srcLocation);
+  #endif
```

Comment:
Looks like you are deleting the test case including backslashes? Don't we want to test both cases on Windows?

Reply:
@smtakeda 
The above code pushes new directory to the vector of directories. And line 64,65 is the old directory pushed to vector. We did not delete we are adding all the directories to the vector and running tests on all. 
Line 74 goes through all directories and runs tests.